exonum/src/runtime/versioning.rs

```diff
@@ -69,6 +69,29 @@
 //!
 //! [semantically versioned]: https://semver.org/
 //!
+//! ## Transactions versioning
+//!
+//! To be able to process transactions, service must have a static mapping between numeric
+//! identifier of transaction and logic of transaction processing. Logic of transaction processing
+//! may include deserializing input parameters from byte array, processing the input and reporting
+//! the execution result (which can be either successful or unsuccessful).
+//!
+//! **Important:** Transaction numeric identifier is considered a constant during all the time of
+//! service existence. It means that if transaction was declared with certain ID, its logic can
+//! be updated (e.g., to fix a bug) or be removed, but it **never** should be replaced with other
+//! transaction.
+//!
+//! If transaction was removed from service, attempt to invoke it should always
+//! result in returning an `ExecutionError`.
+//!
+//! You should use [`CommonError::MethodRemoved`] to report the error in case a method was removed.
```

Comment:
This isn't actually a link, is it? You should use the full name (`CommonError::MethodRemoved`) in the link below, and probably link to the variant (via `#variant.MethodRemoved`).